float64 pos1
int32 pos2
---
bool ret
